run.py

```diff
@@ -372,15 +369,21 @@ def reindex_failed(app):
                         default=False,
                         help='sends bibcodes to augment affilation pipeline, works with --filename')
     parser.add_argument('--solr-collection',
-                       dest='solr_collection',
-                       default='collection2',
-                       action='store',
-                       help='name of solr collection, currently only used by rebuild collection')
+                        dest='solr_collection',
+                        default='collection2',
+                        action='store',
+                        help='name of solr collection, currently only used by rebuild collection')
     parser.add_argument('-x',
                         '--rebuild-collection',
                         action='store_true',
                         default=False,
                         help='Will send all solr docs for indexing to another collection; by purpose this task is synchronous. You can send the name of the collection or the full url to the solr instance incl http via --solr-collection')
+    parser.add_argument('--priority',
+                        dest='priority',
+                        action='store',
+                        default=0,
+                        type=int,
+                        help='priority to use in queue, typically cron jobs use a high priority.  preferred values are 1 to 10 where 10 is the highest priority')
```

Comment:
Small inconsistency, the messages says that the preferred values do not include zero but we have zero as default value all over the code. Maybe we can just say from 0 to 10?